module-sys/SystemManager/stats/stats/Sink.hpp

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#pragma once
+
+#include <cstddef>
+
+class Sink
+{
+  public:
+    virtual ~Sink()                                 = default;
+    virtual void put(const char *data, size_t size) = 0;
```

Comment:
Consider replacing data/size pair args with std::span.

Reply:
but do we support C++20? :D

Reply:
Good point. https://en.cppreference.com/w/cpp/compiler_support says  std::span was introduced in GCC10, so it should work. We might not have C++20 enabled in CMake, though.

Reply:
I tried to enable the c++20 support, but it seems that some parts of the nxp fsl library use deprecated constructs(`compound assignment with 'volatile'-qualified left operand is deprecated [-Werror=volatile]`). Of course, it can be easily fixed, but I believe such actions are out of the scope of this PR.